### DISCUSSION TITLE
Renovate doesn't combine all deps with newer versions into a `group:allNonMajor` PR

### DISCUSSION THREAD

**Discussion by mcmxcdev** [renovatebot/renovate#19193] [Request Help] [answered]
I am not sure if renovate is to blame here or if we have a misconfigured configuration, so it would be great to get some help.

What we want is to: 
- delay major version updates by 1 month -> `stabilityDays`
- upgrade all-non major versions grouped as one PR weekly

So the "Pending Status Checks" part looks 100% correct, but the "Awaiting schedule" part here only shows upgrades for `nx` monorepo instead of all the deps in our `package.json` that qualify for a version upgrade (patch or minor).  

Is there a limit to the maximum number of dependencies that can be upgraded by renovate in a single PR?

Or what seems to be the issue here?

***

![Screenshot from 2022-11-30 15-09-32](https://user-images.githubusercontent.com/16797721/204817615-5d82c516-e004-42fd-b033-d8cc66204998.png)

***

This is our config:

```json
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": [
    "config:base",
    "schedule:weekly",
    "helpers:disableTypesNodeMajor",
    "group:allNonMajor"
  ],
  "labels": ["dependencies"],
  "enabledManagers": ["npm", "github-actions", "dockerfile", "docker-compose"],
  "ignoreDeps": [
    "googleapis",
    "node",
    "react-phone-number-input",
    "pnpm",
    "postgres"
  ],
  "packageRules": [
    {
      "matchUpdateTypes": ["major"],
      "stabilityDays": 30
    },
    {
      "matchPackagePatterns": ["mantine"],
      "groupName": "mantine"
    },
    {
      "matchPackagePatterns": ["temporalio"],
      "groupName": "temporalio"
    }
  ]
}

```

**Comment by viceice** [comment, marked as answer]
you probably use ranges, which aren't changed until the new version falls outside by default

- https://docs.renovatebot.com/configuration-options/#rangestrategy

**Reply by mcmxcdev** [reply]
Oh wow, that's probably it! So in our case, we should probably switch `rangeStrategy` to use `bump` rather than the default `replace` ?

**Reply by viceice** [reply]
depends on what update do you like to have 😉

**Reply by mcmxcdev** [reply]
Works like a charm now! Thanks again for your help!
